src/library/coverartcache.cpp

```diff
@@ -48,6 +48,7 @@ CoverArtCache::~CoverArtCache() {
 QPixmap CoverArtCache::requestCover(const CoverInfo& requestInfo,
                                     const QObject* pRequestor,
                                     const int desiredWidth,
+                                    const double devicePixelRatio,
```

Comment:
Note that the devicePixelRatio should be specified separately for each widget that access CoverArtCache. I thought about setting a global scale factor for CoverArtCache, but that would not work well in case widgets on different screens with different scale factors access the CoverArtCache singleton.